torch2jax/__init__.py

```diff
@@ -101,6 +101,8 @@ def dtype(self) -> torch.dtype: return j2t_dtype(self.value.dtype)
   def ndim(self) -> int: return len(self.value.shape)
   @property
   def shape(self): return self.value.shape
+  @property
+  def device(self): return torch.device("cpu")
```

Comment:
This is fishy to me. Why should we return "cpu" here as opposed to "cuda:0" or anything else?